appveyor.yml

```diff
@@ -1,27 +1,33 @@
 version: 0.0.10.{build}-AppVeyor
-image: Visual Studio 2015
+image: Visual Studio 2017
 configuration: Release
+
 assembly_info:
   patch: true
   file: '**\AssemblyInfo.*'
   assembly_version: '{version}'
   assembly_file_version: '{version}'
   assembly_informational_version: '{version}'
+
```

Comment:
I don't believe this will patch projects using the new SDK. You might need to set the version in the csproj like so: https://github.com/WiseTechGlobal/WTG.Analyzers/blob/master/build/Global.props#L22